.github/workflows/cspell-problem-matcher.json

```diff
@@ -0,0 +1,16 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "cspell",
+      "pattern": [
+        {
+          "regexp": "^(.*):(\\d+):(\\d+)\\s+\\-\\s+(.*)$",
```

Comment:
What does this do?

Reply:
Sorry, this shouldn't have been included, it should be part of the other PR in the descritpion